src/sentry/incidents/model_temp.py

```diff
@@ -0,0 +1,999 @@
+from __future__ import annotations
```

Comment:
This is a straight duplicate of the `incidents/models.py` file so we can point getsentry to reference the new file